Userland/Libraries/LibGUI/FileSystemModel.cpp

```diff
@@ -181,10 +180,29 @@ String FileSystemModel::Node::full_path() const
 
 ModelIndex FileSystemModel::index(String path, int column) const
 {
-    LexicalPath lexical_path(move(path));
+    auto maybe_node = node_for_path(move(path));
+    if (maybe_node.has_value()) {
+        return maybe_node.value()->index(column);
+    } else {
```

Comment:
No need for `else` after `return`.

Reply:
Right, I'm used to this from Python. :^)